modules/exploits/multi/http/atlassian_crowd_pdkinstall_plugin_upload_rce.rb

```diff
@@ -0,0 +1,138 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::EXE
```

Comment:
I don't think this is being used.

Reply:
Yeahhhh I have no idea why thats in here either; came with the previous JIRA module that I based this off of and even there I'm not sure why its included. Missed my to-do list when I was cleaning up this module, let me resolve that.

Reply:
Resolved in https://github.com/rapid7/metasploit-framework/pull/15501/commits/27f70af1b39b9b49452f727208975ed1ef518423